api/routes/apds/versions/delete.js

```diff
@@ -0,0 +1,24 @@
+const logger = require('../../../logger')('apd versions route delete');
+const { can, userCanAccessAPD, loadApd } = require('../../../middleware');
+
+module.exports = app => {
+  logger.silly('setting up DELETE /apds/:id/versions route');
+  app.delete(
+    '/apds/:id/versions',
+    can('submit-document'),
+    userCanAccessAPD(),
+    loadApd(),
```

Comment:
We have a collection of Express middlewares to do some of the common access checks and data loading for us, so we don't have that code duplicated all over.  🙂